File: 01-c/problem_set/hello/hello.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    string name = get_string("What's your name? ");
    printf("hello, %s\n", name);
}

// check50 cs50/problems/2022/x/hello
// style50 hello.c
// submit50 cs50/problems/2022/x/hello

File: 06-python/problem_set/sentimental-hello/hello.py
import sys
from cs50 import get_string

# sys.argv[] looks at the command line arguments
# use command-line argument entered after 'python name.py'
# command line entry: 'python name.py Eric' returns 'hello, my name is Eric'

# sys.exit() leaves the interpreter (exits the whole program)

if len(sys.argv) > 1:
    sys.exit("Usage python hello.py")

name = get_string("What is your name? ")

print("hello,", name)
print()